### PR TITLE
Bug 1481166: Add SourceBuffer.changeType

### PR DESCRIPTION
**Pull request by a2sheppy**

### CODE REVIEW

**Comment by a2sheppy** [comment]
Oh yeah, I also set the experimental flags to false, because they're not.

File: api/SourceBuffer.json [hunk 1]
```diff
@@ -277,7 +277,107 @@
...
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "MediaSourceExperimental",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "70"
```
Comment (Elchi3) [inline api/SourceBuffer.json]: Ranges are reverse-chronological, please.
See https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#the-support_statement-object

File: api/SourceBuffer.json [hunk 2]
```diff
@@ -277,7 +277,107 @@
...
```
Comment (Elchi3) [inline api/SourceBuffer.json]: ditto

File: api/SourceBuffer.json [hunk 3]
```diff
@@ -277,7 +277,107 @@
...
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "media.mediasource.experimental.enabled",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              {
+                "version_added": "63"
```
Comment (Elchi3) [inline api/SourceBuffer.json]: ditto

File: api/SourceBuffer.json [hunk 4]
```diff
@@ -277,7 +277,107 @@
...
```
Comment (Elchi3) [inline api/SourceBuffer.json]: ditto

**Pull request merged**